runtime.go

```diff
@@ -200,10 +199,6 @@ func (self *Runtime) Ticks() uint64 {
 	return self.ticks
 }
 
-func (self *Runtime) SetStackDepthLimit(limit int) {
-	self.stackDepthLimit = limit
-}
-
```

Comment:
Since we're making changes inside goja anyway, would we rather keep this and update the condition in `pushCtx` to use it? Would reduce the changes we'd make in `baas` but would make us more divergent from the goja master.

Reply:
[q]I agree with your concern about divergence. Do you think we can still keep it and just add calling `vm.SetMaxCallStackSize(funcexeccommon.DefaultMaxCallStackSize)` in baas? And we only need to change the `panic` in the `pushCtx`.

Reply:
Went with this approach!